### PR TITLE
update the default configuration

### PR DESCRIPTION
**Pull request by laijs**
Signed-off-by: Lai Jiangshan <jiangshanlai@gmail.com>

### CODE REVIEW

File: package/dist/etc/hyper/config [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
-# CBFS coreboot fs for boot image, if this is proviced, kernel and initrd will be ignored
+
+# CBFS coreboot fs for boot image, if it is set, Kernel and Initrd will be ignored
 Cbfs=/var/lib/hyper/cbfs-qboot.rom
-# storage driver for hyper, valid value includes devicemapper, overlay, and aufs
-#StorageDriver=overlay
-# Bridge device for hyper, default is hyper0
-#Bridge=
+
+# Boot cdroom for "vbox" hypervisor (for mac only)
```
Comment (carmark) [inline package/dist/etc/hyper/config]: cdroom -> CDROM?
vbox -> VirtualBox?

File: package/dist/etc/hyper/config [hunk 2]
```diff
@@ -1,21 +1,45 @@
-# Boot kernel for KVM/XEN
+# configurations for hyperd
+
+# Root directory for hyperd
+# Root=/var/lib/hyper/
+
+# Specify the hypervisor: libvirt, qemu, qemu-kvm, kvm, xen (for linux)
+#                         vbox (for mac).
```
Comment (carmark) [inline package/dist/etc/hyper/config]: VirtualBox should work on linux, it does support all platforms.  The hyper may use that when virtualbox is available on any system.

File: package/dist/etc/hyper/config [hunk 3]
```diff
@@ -1,21 +1,45 @@
...
 Cbfs=/var/lib/hyper/cbfs-qboot.rom
-# storage driver for hyper, valid value includes devicemapper, overlay, and aufs
-#StorageDriver=overlay
-# Bridge device for hyper, default is hyper0
-#Bridge=
+
+# Boot cdroom for "vbox" hypervisor (for mac only)
+# Vbox=/opt/hyper/static/iso/hyper-vbox-boot.iso
+
+# storage driver for hyperd, valid value includes devicemapper, overlay, and aufs
```
Comment (carmark) [inline package/dist/etc/hyper/config]: storage -> Storage?

File: package/dist/etc/hyper/config [hunk 4]
```diff
@@ -1,21 +1,45 @@
...
+# StorageDriver=overlay
+
+# Bridge device for hyperd, default is hyper0
+# Bridge=
+
 # Bridge ip address for the bridge device
-#BridgeIP=
+# BridgeIP=
+
 # if the host IP is provided, a TCP port will be listened for, same as the '--host' option
```
Comment (carmark) [inline package/dist/etc/hyper/config]: We may fix this format, this is an old format error.

if -> If

File: package/dist/etc/hyper/config (hunk 1 above)
Comment (laijs) [inline package/dist/etc/hyper/config]: "vbox" hypervisor, "vbox" is one of the driver name

Comment (carmark) [inline package/dist/etc/hyper/config]: fine for me.

**Comment by laijs** [comment]
updated, merge it.

**Pull request merged**